rest/api_collections_test.go

```diff
@@ -269,6 +269,78 @@ func TestMultiCollectionDCP(t *testing.T) {
 	// require.NoError(t, rt.WaitForDoc(docID))
 }
 
+func TestMultiCollectionDynamicChannelAccess(t *testing.T) {
+	base.TestRequiresCollections(t)
+	//base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll)
+
+	if !base.TestUseXattrs() {
+		t.Skip("Test relies on import - needs xattrs")
+	}
```

Comment:
I can't see anything in the test writing directly to the bucket/collection/datastore. It all goes through SG, which makes me think this isn't nessesary.

Reply:
[5b2f320](https://github.com/couchbase/sync_gateway/pull/6009/commits/5b2f3202cb1f09cd0f83b94dd7186af73eebdb63)